dev/package-examples/auditd-2.0.4/auditbeat/config/config.yml

```diff
@@ -0,0 +1,45 @@
+{{ if eq .GOOS "linux" -}}
```

Comment:
@ph @andrewkroh @jfsiii @mattapperson This is one more example on where the configs are currently dynamically generated on the edge based on some options or OS. I wonder how you imagine this to work in the future?

Reply:
Agree, Fleet will need to get this meta data form the agent. @ph I leave it up to you on how this is "communicated".

On the integrations side when no Fleet is around, I expect that in the manifest file there will be a list of predefined options for these params and the user needs to select it.